nowcasting_metrics/database/forecast.py

```diff
@@ -37,13 +40,14 @@ def get_forecast_values(session: Session, model_name: str) -> pd.DataFrame:
 
     forecasts_ids = query.all()
     forecasts_ids = [m.id for m in forecasts_ids]
-    logger.debug("got forecast ids")
+    logger.debug(f"got forecast ids, ({len(forecasts_ids)})")
```

Comment:
maybe change this log to say "got 200 forecast ids" instead of "got forecast ids 200" otherwise it might be misunderstood to mean that's the id of the forecast

Reply:
tahnks, good idea